src/gluonts/time_feature/seasonality.py

```diff
@@ -14,12 +14,14 @@
 import logging
 
 import pandas as pd
+
 from gluonts.time_feature import norm_freq_str
 
 logger = logging.getLogger(__name__)
 
 
 DEFAULT_SEASONALITIES = {
+    "S": 86400,
```

Comment:
This looks very high. I wonder, why do we always want to refer to the same value a day ago?